src/EduManagementLab.Core/Services/CourseService.cs

```diff
@@ -68,6 +67,16 @@ public void DeleteCourse(Guid id)
             _unitOfWork.Courses.Remove(course);
             _unitOfWork.Complete();
         }
+        public Course GetCourseIncludeMemberships(Guid courseId, bool includeMembership, bool includeUser)
```

Comment:
Should it be called GetCourseIncludeMemberships when you can pass a parameter to not include memberships?
I think it can be called GetCourse and take the parameters for including.

Maybe we can have only one parameter? Suggestion: includeMembershipUsers.

Reply:
Renamed Method to GetCourse , And have only includeMembershipUsers & CourseId for parameter,